packages/docs/jest.config.js

```diff
@@ -12,7 +12,10 @@ module.exports = {
     'jsx',
     'node',
   ],
-  moduleNameMapper: { '^.+\\.module\\.(css|sass|scss)$': 'identity-obj-proxy' },
+  moduleNameMapper: {
+    '^.+\\.module\\.(css|sass|scss)$': 'identity-obj-proxy',
+    '^msw-storybook-addon$': '<rootDir>/../msw-addon/dist/index.node.js',
```

Comment:
@yannbf this makes sense just make sure this isn't needed for the end developer who will be using Storybook and the add-on for integration testing.